src/main/java/pl/byczazagroda/trackexpensesappbackend/repository/WalletRepository.java

```diff
@@ -1,11 +1,17 @@
 package pl.byczazagroda.trackexpensesappbackend.repository;
 
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
 import pl.byczazagroda.trackexpensesappbackend.model.Wallet;
 
+import java.util.List;
+
 /**
  * Repository for Wallet Entity.
  * @Since 0.2.0
  */
 public interface WalletRepository extends JpaRepository<Wallet, Long> {
+    @Query("SELECT w FROM Wallet w WHERE w.name LIKE %:name%")
+    List<Wallet> findAllByLikeName(@Param("name") String name);
```

Comment:
You don't have to create your own SQL queries, you can just use query methods provided by Spring JPA.
Here is a referecne: https://docs.spring.io/spring-data/jpa/docs/current/reference/html/#appendix.query.method.subject

Reply:
Yes, you have right, but it was my suggestion to write with SQL to practse this query